api/src/test/java/edu/cornell/mannlib/vitro/webapp/dynapi/RPCEndpointTest.java

```diff
@@ -0,0 +1,114 @@
+package edu.cornell.mannlib.vitro.webapp.dynapi;
+
+import static org.easymock.EasyMock.anyObject;
+import static org.easymock.EasyMock.anyString;
+import static org.easymock.EasyMock.expect;
+import static org.powermock.api.easymock.PowerMock.createMock;
+import static org.powermock.api.easymock.PowerMock.mockStatic;
+import static org.powermock.api.easymock.PowerMock.replay;
+import static org.powermock.api.easymock.PowerMock.verify;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.easymock.annotation.Mock;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.Action;
+import edu.cornell.mannlib.vitro.webapp.dynapi.components.OperationResult;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(ActionPool.class)
+public class RPCEndpointTest {
+
+	final static String URI_TEST = "/api/rpc/test";
+
+	private static RESTEndpoint restEndpoint;
```

Comment:
private static RPCEndpoint rpcEndpoint ????

Reply:
Good catch, I'll get right to this.

Reply:
Fixing this required me to fix many more things.
I practically rewrote the tests and had to merge staging into this branch.
It is now ready for review again.